mapie_v1/conformity_scores/utils.py

```diff
@@ -0,0 +1,22 @@
+from typing import Union
+from mapie.conformity_scores import (
+    BaseRegressionScore,
+    AbsoluteConformityScore,
+    GammaConformityScore,
+    ResidualNormalisedScore,
+)
+
+
+def select_conformity_score(
```

Comment:
We definitely need this 👍
Two comments :
- We should probably use a dict instead of if/else statements
- We should put that dict in a __init__ file inside conformity_scores directory